test/disabled-options.test.js

```diff
@@ -66,3 +66,22 @@ t.test('should not apply the delay header', async t => {
     t.equal(response.headers[HEADERS.delay], undefined)
   })
 })
+
+t.test('should not apply headers if the headers option is false', async t => {
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin, {
+    headers: false
+  })
+  t.teardown(() => fastify.close())
+
+  fastify.get('/', async () => 'Hello from fastify-slow-down!')
+
+  await slowDownAPI(DEFAULT_OPTIONS.delayAfter, () => fastify.inject('/'))
+  const response = await fastify.inject('/')
+
+  t.has(response.headers, {
+    [HEADERS.limit]: undefined,
+    [HEADERS.remaining]: undefined,
+    [HEADERS.delay]: undefined
```

Comment:
It may sound silly, but `notHas` reads easier as in "check that the response **does not have** these headers".

```suggestion
  t.notHas(response.headers, {
    [HEADERS.limit]: String,
    [HEADERS.remaining]: String,
    [HEADERS.delay]: String
```